recipes/pyranha/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.8" %}
+
+package:
+  name: pyranha
+  version: {{ version }}
+
+source:
+  fn: piranha-{{ version }}.tar.gz
+  url: https://github.com/bluescarni/piranha/archive/v{{ version }}.tar.gz
+  sha256: 649f91ceee56446f903551f0665b8807b833d91fd9622c412eadc95d353ac8e6
+
+build:
+  number: 0
+  skip: true        # [win and py3k]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - gmp
+    - mpfr
+    - boost     1.62.*
+    - bzip2     1.0.*
+    - zlib      1.2.*
+    - piranha   {{ version }}
+    - python
+  run:
+    - gmp
+    - mpfr
+    - boost     1.62.*
+    - bzip2     1.0.*
+    - zlib      1.2.*
+    - piranha   {{ version }}
+    - python
+
+test:
+  requires:
+    - mpmath
+    - numpy
```

Comment:
If `mpmath` is used only for conversion, then it need not be added as a runtime requirement. Then `mpmath` should only be a test requirement and `numpy` will be a runtime requirement